web/modules/custom/hook_basic/hook_basic.module

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
```

Comment:
```
/**
 * Implements hook_entity_view().
 */
```

Reply:
Why is this still not addressed ? Along with other feedbacks as well

Reply:
I am not able to understand this feedback.

Reply:
Docblock added for the hook entity view implementation is this as of now

```
/**
 * Function to display number of times a page is visited.
 *
 * @param array $build
 *   An associative array representing the renderable elements of the entity.
 * @param \Drupal\Core\Entity\EntityInterface $entity
 *   The entity object whose view is being built.
 * @param \Drupal\Core\Entity\Display\EntityViewDisplayInterface $display
 *   The display configuration object used to build the entity view.
 * @param mixed $view_mode
 *   The view mode of settings.
 */
```

Which is incorrect, we need to replace it with
/**
 * Implements hook_entity_view().
 */

Reply:
ok got it